### PR TITLE
Implement change in questions to select a business model 

### PR DESCRIPTION
**Pull request by Bachibouzouk**
Do not forget to run `python manage.py migrate business_model` before trying it locally

This version is already updated online

### CODE REVIEW

File: app/business_model/models.py [hunk 1]
```diff
@@ -57,6 +57,7 @@ def default_fate_values(self):
 
 class BMQuestion(models.Model):
     question_for_user = models.TextField(null=False)
+    sub_question_to = models.IntegerField(null=True)
```
Comment (paulapreuss) [inline app/business_model/models.py:60]: This feature is not being used yet, right? Or am I missing it?

**Comment by paulapreuss** [comment]
Functionality looks good. A few things I noticed while clicking through the questionnaire:
- If community-led is not suitable, this text comes up
> This project might not be suited for a community-led model based on your answers to the questionnaire. Please manually select your desired business model.

Since now we only have one alternative to the community led, and it is automatically selected when the community-led is not an option, we should probably change this text.
- Some questions don't have consistent formatting in the answers (e.g. 
image
- This nested question in financial capacities should have an option for None (for the case that No was selected in the previous question, since we are not doing the sub-questions yet)
image

**Review by paulapreuss — APPROVED** [review]

File: app/business_model/models.py (hunk 1 above)
Comment (Bachibouzouk) [reply app/business_model/models.py:60]: indeed it is not used yet, only for future use

**Comment by Bachibouzouk** [comment]
> * This nested question in financial capacities should have an option for None (for the case that No was selected in the previous question, since we are not doing the sub-questions yet)

Actually, the lowest answer already provide the score 0, so it is equivalent to None

**Pull request merged**